tests/unit/dev_support.py

```diff
@@ -54,13 +54,17 @@ def _wet(func):
 
 
 # Actual test cases:
-def test_dry_function(): _dry(_function)
+def test_dry_function():
+    _dry(_function)
```

Comment:
Can you explain why you changed these? Not really a problem, just doesn't seem to have anything to do with this PR, and it doesn't really seem like an improvement (though not really worse either).